package.json

```diff
@@ -45,32 +46,28 @@
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.4.5",
-    "@babel/preset-env": "^7.4.5",
-    "@babel/preset-react": "^7.0.0",
     "@types/escape-html": "^1.0.0",
     "@types/is-hotkey": "^0.1.1",
-    "@types/jest": "^28.1.4",
     "@types/lodash": "^4.14.137",
     "@types/react": "^18.0.15",
     "@types/react-modal": "^3.10.6",
     "@typescript-eslint/eslint-plugin": "^5.30.5",
     "@typescript-eslint/parser": "^5.30.5",
     "@vitejs/plugin-react": "^2.0.0",
+    "c8": "^7.12.0",
```

Comment:
Some people just want to watch the world burn. What a dump package name 🙃 I hate having to google every package...